### PR TITLE
Update Archiware P5 to 13.2 release

### PR DESCRIPTION
**Pull request by IgorC-test**
Thank you for your submission to the FreeNAS community plugins repository!

Please ensure the following guidelines are met when submitting a new Plugin.

1. Add the entry to INDEX in alphabetical order
2. Includes a new icon in the ./icon/ directory
3. Ensure the submitted icon file is a valid PNG that is 128x128 in size
4. Add new plugin cirrus task

### CODE REVIEW

**Comment by IgorC-test** [comment]
Hello,
Could You please review and approve update of Archiware P5 to 13.2 Release, we have some customers that are on TrueNAS 13.2 and they cannot install P5 until this change is approved
Much appreciated

**Comment by tprelog** [comment]
I can't help with the approval of this PR but you should be thinking about moving away from the plugin and simply installing Archiware P5 into a standard jail. As a former plugin maintainer, I've waited almost 6 months for changes to merge and make their way onto TrueNAS. I was constantly having to write workarounds into my plugin's installation scripts. Also, ix-systems is completely dropping support for plugins in 2025.

See the following links for details

https://www.truenas.com/community/threads/ixsystems-when-are-you-going-to-stop-pretending-that-core-has-usable-plugins.104355/

https://www.truenas.com/blog/the-future-of-truenas-plugins-is-apps/

**Review by sonicaj — APPROVED** [review]

**Pull request merged**
